Test/FMI3/parser_test_xmls/model_structure/valid/modelDescription.xml

```diff
@@ -10,11 +10,13 @@
 <ModelVariables>
     <Float64 name="state_var" valueReference="0"/>
     <Float64 name="der(state_var)" valueReference="1" causality="output" variability="continuous" initial="approx" derivative="1" start="1.0"/>
+    <Float64 name="output_var" valueReference="2" causality="output"/>
+    <Float64 name="init_unknown_var" valueReference="3"/>
```

Comment:
This is a local variable. They're not considered InitialUnknown right?

Reply:
Fixed, also with error check on this now.

Reply:
Right. Would require initial="calculated" or similar. Removed it for simplicity here and added the derivative as InitialUnknown instead (which is also required).